### PR TITLE
Milestone 1: Data architecture documentation

### PR DESCRIPTION
**Pull request by gango-anan**
# In this milestone, I;
- Created an ERD to model the architecture of the application
- Saved the ERD as a pdf file in a folder called docs in the application repository.

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @gango-anan  :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this milestone can be approved.

## REQUIRED

* [ ] The Primary Keys and Foreign Keys are not present in your tables. Make sure that Primary Keys (PK) and Foreign Keys (FK) are assigned correctly.


### Make the required change and request another review. 
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by MahmoudBakr23 — APPROVED** [review]
# **Approved ✔✔**

<hr>

Hello there, great job creating the ERD, this is your very first step into a very fun journey of building the ror-social-media rails app. So, good luck and happy coding 😊👨‍💻💻

Go ahead and merge to the development branch, and move on to the next milestone.

**Pull request merged**
